### PR TITLE
Report when consoles are disconnected

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
This will help debugging issues like when post_fail_hook is triggered
after disconnecting consoles on remote SUT's.

- Related ticket: https://progress.opensuse.org/issues/62366
- Verification run:
  - x86_64: https://openqa.suse.de/tests/4029409#step/setup/11
  - s390-kvm: all workers broken :(

### CODE REVIEW

**Comment by foursixnine** [comment]
Isn't this the same pr from couple of days ago?

**Comment by SergioAtSUSE** [comment]
> Isn't this the same pr from couple of days ago?

The same purpose. Slightly different content.
It doesn't make mentioned test to fail.

I have just seen the travis failure. I don't understand what is wrong.

**Comment by foursixnine** [comment]
@SergioAtSUSE Do you need some help with this?

**Comment by SergioAtSUSE** [comment]
> @SergioAtSUSE Do you need some help with this?

No, I just didn't work again on this. Do you want to take over?
If not, I will after I finish the changes for *network_configuration* test suite

**Comment by foursixnine** [comment]
> > @SergioAtSUSE Do you need some help with this?
> 
> No, I just didn't work again on this. Do you want to take over?
> If not, I will after I finish the changes for _network_configuration_ test suite

The original ticket is rejected, if you wish open a ticket in the team backlog explaining the idea and maybe we can pick it up at some point?.

**Comment by SergioAtSUSE** [comment]
I don't know how to solve this: https://travis-ci.org/github/os-autoinst/os-autoinst-distri-opensuse/jobs/692563937

File: lib/power_action_utils.pm [hunk 1]
```diff
@@ -49,21 +49,28 @@ $vnc_console get required variable 'SVIRT_VNC_CONSOLE' before assignment.
...
+            diag('prepare_system_shutdown: (iucvconn) ignoring already shut down console') if ($@);
         }
         console('installation')->disable_vnc_stalls;
+        record_info('installation', 'prepare_system_shutdown: (installation) VNC stall is disabled');
     }
 
     if (check_var('VIRSH_VMM_FAMILY', 'xen') || get_var('S390_ZKVM')) {
         my $vnc_console = get_required_var('SVIRT_VNC_CONSOLE');
         console($vnc_console)->disable_vnc_stalls;
+        record_info("$vnc_console", "prepare_system_shutdown: ($vnc_console) VNC stall is disabled");
```
Comment (foursixnine) [inline lib/power_action_utils.pm:71]: I'm guessing this is the culprit, perhaps $vnc_console is undef at this point (instead of empty)

**Comment by foursixnine** [comment]
@SergioAtSUSE do you plan to work on this? if not, create a ticket and remove the not ready tag, so that stale bot takes care of it when the time is up

**Comment by SergioAtSUSE** [comment]
@okurz, I started using diag and was asked to use `record_info`.

**Comment by okurz** [comment]
sorry, I can't find any comment regarding "please use `record_info`"

**Comment by foursixnine** [comment]
Closed due to inactivity for almost 6 months. Please reopen when you plan to work on it.

**Pull request closed without merging**